config/settings_local.py

```diff
@@ -70,7 +70,7 @@
             "gssencmode": "disable",
             # this is a hack due to our inability to set a custom parameter either at
             # the database or role level in managed databases such as AWS RDS
-            "options": "-c osidb.acl=''",
+            "options": "-c osidb.acl=00000000-0000-0000-0000-000000000000",
```

Comment:
I trust you this is correct but could you please explain what this value is? :smile:

Reply:
It's the UUID equivalent of `NULL` or empty `''` [1]

[1] https://en.wikipedia.org/wiki/Universally_unique_identifier#Nil_UUID

Reply:
Oh I see! Some Postgres magic :smile_cat: